en/developer-docs/docs/administer/configure-approvals-for-choreo-workflows.md

```diff
@@ -61,3 +72,7 @@ Once you enable the approval process for a workflow, see the following details o
 === "API subscription"
 
      Once you configure an approval process for API subscription, administrators can select the **Approval required** checkbox to create or update subscription plans to require approval. For details, see [Create API Subscription Plans](../administer/create-api-subscription-plans.md). API consumers using these plans must request approval to proceed. For details, see step 7 in [Subscribe to an API with a Subscription Plan](../api-management/manage-api-traffic/subscribe-to-an-api-with-a-subscription-plan.md). An authorized approver must then [review and approve the request](./review-workflow-approval-requests.md) before the subscription is granted.
+
+=== "URL customization"
+
+     Once you configure an approval process for URL customization, developers must submit a request for approval when configuring a custom URL for a component. Instead of being automatically deployed, the URL mapping will go into a pending state. An authorized approver must then [review and approve the request](./review-workflow-approval-requests.md) before the custom URL is activated. 
```

Comment:
```suggestion
     Once you configure an approval process for URL customization, developers must submit a request for approval to configure a custom URL for a component. Instead of being automatically deployed, the URL mapping will go into a pending state. An authorized approver must then [review and approve the request](./review-workflow-approval-requests.md) before the custom URL is activated for the respective component. 
```
Suggested some changes. Check and incorporate if applicable.

Reply:
addressed with fa21322624c9540ef207ec3ef0ef7a59a2024e9b